setup.py

```diff
@@ -0,0 +1,18 @@
+from distutils.core import setup
+
+execfile('Tribler/Core/version.py')
```

Comment:
Ah, but AFAICS that's not an issue in our case as that file is not importing anything at all, just setting a few vars. If the file was importing stuff, that might be a problem.

I don't really care either way, I was just curious :)